.github/dependabot.yml

```diff
@@ -2,6 +2,7 @@ version: 2
 updates:
   - package-ecosystem: "composer"
     directory: "/"
+    versioning-strategy: lockfile-only
```

Comment:
Even if they don't break tests and weren't responsible for this near-regression in the first place, the idea of unattended major updates makes me shudder. Plus it becomes difficult to pin dependencies.